assets/src/blocks/Counter/CounterFrontend.js

```diff
@@ -21,6 +21,10 @@ export class CounterFrontend extends Component {
     if (completed_api && completed_api.startsWith('https://')) {
       window.addEventListener('updateCounter', function () { counter.calculateRemaining(); }, false);
     }
+    // Append the counter inside the En-Form
+    var counterBar = document.querySelector('.counter-style-en-forms-bar');
+    var enFormHeader = document.querySelector('.enform-extra-header-placeholder');
+    enFormHeader.append(counterBar);
```

Comment:
Tested with a counter with different style, you should check for at least the existence of the counter `.counter-style-en-forms-bar` before appending it, if you don't it adds null :grin: 
![Screenshot from 2020-12-10 14-32-49](https://user-images.githubusercontent.com/617346/101778850-bc44f880-3af4-11eb-8173-64eeb80f6636.png)